en/identity-server/next/docs/references/conditional-auth/api-reference.md

```diff
@@ -965,3 +1020,64 @@ Contains the authentication step information. It may be a null or invalid step n
       <td>This is the device property that is extracted from the raw userAgent string.</td>
     </tr>
   </table>
+
+You can securely store consumer keys and secrets as **secrets** in conditional authentication scripts and refer to
+them in your conditional authentication scripts using the `secrets.key` syntax. For example, to retrieve a secret value, you may use:
+```angular2html
+var consumerSecret = secrets.clientSecret;
+```
+For more information on adding secrets, refer to the [Add a secret to the script]({{base_path}}/guides/authentication/conditional-auth/configure-conditional-auth/#add-a-secret-to-the-script) section in the 
+documentation.
+
```

Comment:
Refer to my comment in other doc PRs that propose the same

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4772/commits/cf4fb92a77bd7856736c909b594f1ff84eb6c9a0